empack/file_packager.py

```diff
@@ -215,12 +215,10 @@ def split_pack_environment(
                     extend_js(os.path.join(pack_outdir, f"{pkg_outname}.{i}.js"))
 
     # create the base js file
-    lines = []
-    for js_file in js_files:
-        # lines.append(f"    promises.push(import('./{js_file}'));")
-        lines.append(f"await import('./{js_file}')")
-        lines.append(f"await {export_name}._wait_run_dependencies()")
-    txt = "\n".join(lines)
+    txt = f"""
+    importScripts('{"', '".join(js_files)}');
```

Comment:
@martinRenou since the function within we are calling `importScript` is async, we could keep the "await wait_run_dependencies` (not sure if I am over-careful here and this is not needed or if we actually need this)

Reply:
I see. It will rework the code so that it fixes it. So we should do the following?

```js
importScript("libA.js");
await {export_name}._wait_run_dependencies();
importScript("libB.js");
await {export_name}._wait_run_dependencies()
```

Reply:
so with that, we are doing "the right thing" but might be a bit slower. I think we should keep it!